module-apps/apps-common/widgets/DateAndTimeStyle.hpp

```diff
@@ -11,7 +11,7 @@ namespace style::window::date_and_time
     inline constexpr auto separator        = 30;
     inline constexpr auto time_input_12h_w = 120;
     inline constexpr auto time_input_24h_w = 195;
-    inline constexpr auto hBox_h           = height - 1.25 * topMargin;
+    inline constexpr auto hBox_h           = static_cast<int>(height - 1.25 * topMargin);
```

Comment:
What about add `f` to `1.25f`?

Reply:
It probably won't change runtime computational complexity coming from float vs. double operations as all of the values involved in this equation are `constexpr`'s, so the result shall be computed at compilation time, but I've changed it to 1.25f as you suggested ;)